src/Model/Item.php

```diff
@@ -3440,8 +3439,8 @@ public static function prepareBody(array &$item, bool $attach = false, bool $is_
 				$filter_reasons[] = DI::l10n()->t('Content from %s is collapsed', $item['author-name']);
 			}
 
-			if (!empty($item['content-warning']) && (!$uid || !DI::pConfig()->get($uid, 'system', 'disable_cw', false))) {
-				$filter_reasons[] = DI::l10n()->t('Content warning: %s', $item['content-warning']);
+			if ($item['sensitive'] && (!$uid || !DI::pConfig()->get($uid, 'system', 'disable_cw', false))) {
```

Comment:
Quid of the `system. display_sensitive` pconfig key?

Reply:
?